README.md

```diff
@@ -19,10 +19,10 @@ A polyfill for the `{{on}}` element modifier specified by
 ember install ember-on-modifier
 ```
 
-#### Compatibility
+## Compatibility
 
-- Ember.js v2.18 or above
-- ember-cli v2.13 or above
+* Completely inert when running `ember-source` 3.9 or higher
```

Comment:
If I am not mistaken, this should be 3.11.

https://github.com/buschtoens/ember-on-modifier/blob/1816d4c7b3e17aea8ba2ca8208a18aafffb5e556/index.js#L15

Reply:
Yes, you're right my bad.

Reply:
I've amend my commit to fix it